packages/react-core/src/components/Modal/examples/Modal.md

```diff
@@ -13,84 +13,115 @@ import HelpIcon from '@patternfly/react-icons/dist/esm/icons/help-icon';
 
 ## Examples
 
-### Basic
+### Basic modals
+
+Basic modals give users the option to either confirm or cancel an action. To flag an open modal, use the `isOpen` property. To execute a callback when a modal is closed, use the `onClose` property.
 
 ```ts file="./ModalBasic.tsx"
 ```
 
-### With description
+### Scrollable modals
+
+To enable keyboard-accessible scrolling of a modal’s content, pass `tabIndex={0}` to the `<Modal>`.
+
+```ts file="ModalWithOverflowingContent.tsx"
+```
+
+### With a static description
+
+To provide additional information about a modal, use the `description` property. Descriptions are static and do not scroll with other modal content.
 
 ```ts file="./ModalWithDescription.tsx"
 ```
 
 ### Top aligned
 
+To override a modal's default center alignment, use the `position` property. In this example, `position` is set to "top", which moves the modal to the top of the screen.
```

Comment:
Just an aside, I noticed in the docs that the "type" for the `position` prop is "literal", so you don't get a list of alternate positions, even though there is only one available currently ("top"). 

https://patternfly.org/v4/components/modal#modal

https://github.com/patternfly/patternfly-react/blob/122f28feeb8a8130c8beb9e2504ba34e48b911b1/packages/react-core/src/components/Modal/Modal.tsx#L58

Should we update that so that we list the available options? cc @tlabaj

Reply:
I imagine this was for flexibility incase other positions are introduced.

Reply:
@tlabaj nope, there are no other position options. I think a prop for position is good, I was just referring to the docs - the way the prop's value is set to "top" now, the "type" column in the docs shows as `literal`. I think it might help to show it as `'top' | 'default'`, like the `variant` prop's docs.

**position**
<img width="809" alt="Screenshot 2023-01-09 at 6 13 48 PM" src="https://user-images.githubusercontent.com/35148959/211433444-05825e85-a730-477e-b792-15d38bf1ad15.png">

**variant**
<img width="608" alt="Screenshot 2023-01-09 at 6 14 08 PM" src="https://user-images.githubusercontent.com/35148959/211433445-e0ad134b-4a8d-449d-a96f-52ee64124322.png">

Reply:
@mcoker we can open an issue for to add `default` to the type.

Reply:
issue opened: https://github.com/patternfly/patternfly-react/issues/8523